blocks/init/src/Blocks/components/quote/quote.php

```diff
@@ -34,7 +34,7 @@
 $quoteAuthorUse = Components::checkAttr('quoteAuthorUse', $attributes, $manifest);
 ?>
 
-<figure class="<?php echo esc_attr($quoteClass); ?>">
+<figure class="<?php echo esc_attr($quoteClass); ?>" role="presentation">
```

Comment:
Can you paste an example how the quote is announced in voiceover?

Reply:
https://github.com/infinum/eightshift-frontend-libs/assets/97440314/f3216eab-3e42-43b8-ad96-0e7c2902e9c8

In Mozilla Firefox, it is read as 'Figure' and does not completely read the content.